packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { Text, Icon, Money } from '@deriv/components';
+import { TTradingPlatformAccounts, TCFDDashboardContainer, TCFDsPlatformType } from '../Components/props.types';
```

Comment:
```suggestion
import { TTradingPlatformAccounts, TCFDDashboardContainer, TCFDsPlatformType } from 'Components/props.types';
```
can we use alias here

Reply:
We are already using it @hamza-deriv  . We are using "Components".

Reply:
what with the `../Components`?
should not we remove the `..` before the components?
Maybe i am wrong